### ISSUE TITLE
input type range to optionally bind to array of integers to render more than one handle on a slider

### ISSUE THREAD

**Issue by richardhulbert** [bootstrap-vue/bootstrap-vue#5568]
### Is your feature request related to a problem? Please describe...

We sometimes want to be able to have more than one point on the same range slider for instance picking a min and max value on a slider.

### Describe the solution you'd like

Instead of biding the value to an integer it can optionally bind to an array of integers. Each integer would then be rendered as a separate handle for the user to slide.

In effect the component would render multiple inputs super imposed range inputs with the track hidden using css

### Describe alternatives you've considered

I suppose I could wrap the existing <b-input type="range> tags in a parent component that would perform the CSS to hide the subsequent track and progress.

**Comment by jackmu95** [comment]
Since the HTML 5 spec form `<input type="range">` doesn't support multiple handles this would end up as a custom component.

**Comment by richardhulbert** [comment]
Yes - I have started to make one effectively layering ghost inputs on top of each other I only have need of min and max so my implementation is limited atm - I will see what space I have to implement something a bit better before sharing it.
